resources/local/config.toml

```diff
@@ -127,33 +127,20 @@ event_stream_buffer_length = 100
 # If unset, the path must be supplied as an argument via the CLI.
 path = '../node-storage'
 
-# Optional maximum size of the database to use for the block store.
-#
-# If unset, defaults to 483,183,820,800 == 450 GiB.
+# Maximum size of the database to use for the block store.
 #
 # The size should be a multiple of the OS page size.
-#max_block_store_size = 483183820800
+max_block_store_size = 483_183_820_800
```

Comment:
Please add back into the comment the human-readable form of these values.  If you're worried about the comment going out of date, feel free to phrase it like:
`# 483_183_820_800 == 450 GiB.` so at least it won't be as misleading if the actual set value changes.